jme3-examples/src/main/java/jme3test/water/TestPostWater.java

```diff
@@ -277,14 +277,12 @@ public void simpleUpdate(float tpf) {
         water.setWaterHeight(initialWaterHeight + waterHeight);
         if (water.isUnderWater() && !uw) {
 
-            waves.setDryFilter(new LowPassFilter(0.5f, 0.1f));
+           waves.setReverbEnabled(false);
             uw = true;
         }
         if (!water.isUnderWater() && uw) {
             uw = false;
-            //waves.setReverbEnabled(false);
-            waves.setDryFilter(new LowPassFilter(1, 1f));
-            //waves.setDryFilter(new LowPassFilter(1,1f));
+            waves.setReverbEnabled(true);
```

Comment:
Interesting, so it is no longer necessary to continuously change the 'dryFIlter'?

Reply:
I tested it, and it works. @bob0bob care to comment?